pkg/build/apiserver/registry/buildlog/rest_test.go

```diff
@@ -10,15 +11,19 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/watch"
 	apirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/apiserver/pkg/registry/rest"
+	"k8s.io/client-go/informers"
+	informerscorev1 "k8s.io/client-go/informers/core/v1"
 	"k8s.io/client-go/kubernetes/fake"
-	clientgotesting "k8s.io/client-go/testing"
+
+	"k8s.io/client-go/tools/cache"
```

Comment:
nit - no line break here so that k8s.io imports are grouped together. `go fmt` will sort them alphabetically.

```suggestion
	"k8s.io/client-go/tools/cache"
```